sysutils/syslog-ng/Portfile

```diff
@@ -1,61 +1,70 @@
-PortSystem 1.0
-PortGroup  github 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-github.setup	syslog-ng syslog-ng 3.38.1 syslog-ng-
-categories		sysutils
-license			{GPL-2+ OpenSSLException}
-maintainers		nomaintainer
-description		Highly configurable syslog replacement
-long_description	syslog-ng is the world's most flexible and scalable audit \
-					trail processing tool for organizations of any size. It \
-					provides a centralised, securely stored log of all devices \
-					on your network, whatever platform they run on. And \
-					syslog-ng also incorporates a host of powerful features, \
-					including filtering based on message content, as well as \
-					customisable data mining and analysis capabilities.
+PortSystem          1.0
+PortGroup           github 1.0
 
-homepage			https://www.syslog-ng.com/
-github.tarball_from			releases
-checksums			rmd160 cd6e9b67b73a68b126dce8653dbf696583b8d86b \
-			sha256 5491f686d0b829b69b2e0fc0d66a62f51991aafaee005475bfa38fab399441f7
+github.setup        syslog-ng syslog-ng 3.38.1 syslog-ng-
```

Comment:
Nitpick, but can we add a `revision` field while we're at it?

Reply:
@herbygillot Done